src/modules/Packages.rb

```diff
@@ -2702,6 +2700,16 @@ def format_missing_resolvables(type, list)
         _("These items (%{type}) need to be selected to install: %{list}") % {type: type, list: list}
       end
     end
+
+    def fs_mountpoint(filesystem)
+      filesystem.mountpoints[0] || ""
```

Comment:
is it correct behavior if there is more mountpoints for given filesystem? E.g. what happen when it is btrfs with a lot of subvolumes and first one is e.g. "/var/run" and not "/" ? Or does it expect that first one is always main mount-point without subvolumes? and what if one device is mounted to multiple places? does it compute correctly free space?